packages/core/src/DdSdkReactNative.tsx

```diff
@@ -50,7 +51,6 @@ export class DdSdkReactNative {
 
     private static wasAutoInstrumented = false;
     private static features?: AutoInstrumentationConfiguration;
-    private static _isInitialized = false;
```

Comment:
can you please elaborate where cyclic dependency comes from? Normally we can rely on the native SDK `Datadog.isInitialized` method to read the state.

Reply:
**DdRum.ts**
```typescript
import { DdSdkReactNative } from '../DdSdkReactNative';
```

**DdSdkReactNative.ts**
```typescript
import { DdRum } from './rum/DdRum';
```

Reply:
**Some more context:**

`DdRum` depends on `DdSdkReactNative`, which depends on `DdRum`.

With this PR I have moved the attribute to a third class, in order to resolve the cyclic dependency.

I would prefer to keep the parameter on the React layer instead of using the native SDK method. It could be unnecessarily complex to do otherwise, due to the fact that native method calls have to be async in order to be fully compatible with pre new-architecture RN versions.